incubator/mariadb-cluster/values.yaml

```diff
@@ -1,6 +1,6 @@
 ## Bitnami MariaDB image
```

Comment:
I see a different format on other images:

```
image:
  registry: docker.io
  repository: bitnami/XXXX
  tag: Y.Y.Y
  ## Specify a imagePullPolicy
  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
  ##
  pullPolicy: Always
  ## Optionally specify an array of imagePullSecrets.
  ## Secrets must be manually created in the namespace.
  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
  ##
  # pullSecrets:
  #   - myRegistrKeySecretName
```

Reply:
+1, remember to update the master-deploy and slave-statefukset files to retrieve the `image` and `imagePullPolicy` using the new format